deeprankcore/query.py

```diff
@@ -189,15 +189,27 @@ def add(self, query: Query, verbose: bool = False, warn_duplicate: bool = True):
             _log.info(f'Adding query with ID {query_id}.')
 
         query_id_base = query_id.split("_")[0]
+        
+        warn_duplicate = False    
```

Comment:
`warn_duplicate` is a parameter for the user to determine if printing warnings in case of duplicates or not. As you modified it, warnings will be printed independently from what the user has set up.